src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -361,26 +365,32 @@ int countContainersInProgress() {
                         t.getImage(), label, getDisplayName());
 
                 final CompletableFuture<Node> plannedNode = new CompletableFuture<>();
-                r.add(new NodeProvisioner.PlannedNode(t.getDisplayName(), plannedNode, t.getNumExecutors()));
+                DockerNodeFactory.DockerPlannedNode pn =
```

Comment:
`pn` should be declared `final` unless it needs to be mutable ... and it doesn't look like it needs to be mutable.